code/data.php

```diff
@@ -62,6 +62,7 @@
     '<a href="http://python-social-auth-docs.readthedocs.io/en/latest/"> Python Social Auth</a> is an OAuth and OAuth2 client for a multitude of services.',
     '<a href="https://github.com/lepture/flask-oauthlib">Flask-OAuthlib</a> is an OAuth2 Client/Provider for Flask built upon <a href="https://github.com/oauthlib/oauthlib">oauthlib</a>',
     '<a href="https://github.com/lepture/authlib">Authlib</a> has built-in OAuth 2 client for Flask and Django.</a>',
+    '<a href="https://github.com/requests/requests-oauthlib">Requests-OAuthlib</a> has first-class OAuth library support for <a href="http://python-requests.org">Requests</a>.</a>',
```

Comment:
Calling it "first-class" sounds a little bit too much like marketing copy, if you ask me.

Reply:
@jricher the text comes from the repository itself actually: https://github.com/requests/requests-oauthlib

Otherwise what do you think we can put as text?

Reply:
I'm not at all surprised that the original repository would have marketing language like that, but I don't think it's appropriate to copy that here just because it's there. What if a library claimed to be "the best implementation ever of all time" on its repository? 

I would recommend a simple change like: 

```suggestion
    '<a href="https://github.com/requests/requests-oauthlib">Requests-OAuthlib</a> has OAuth library support for <a href="http://python-requests.org">Requests</a>.</a>',
```